its/plugin/projects/cpd/file3.py

```diff
@@ -0,0 +1,28 @@
+class A:
+    def func1():
```

Comment:
we lack a case with code with multiple indentation still being detected correctly. Please add a few lines of code, if possible with multiple level of indentation.